### DISCUSSION TITLE
Status line disappear on printing start

### DISCUSSION THREAD

**Discussion by alxsfrm** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2665] [General]
Hello,
I build the firmware by myself.
I have latest Marlin on SKR mini E3 + TFT35 V3 E3.
I have following issue: before printing i can see in the middle of the screen the status line with extruder XYZ values, but when i'm starting to print the status line disappear, and instead of it there is progress bar.
Also the layer number/height is not shown, just ------ .
How can i make the extruder position and the layer height to be visible during printing ?
Thank you.

**Comment by kisslorand** [comment]
>but when i'm starting to print

"starting to print" is vague... From where? How? (from TFT SD, from TFT USB, from onboard SD, from onboard USB, from OctoPrint connected to TFT, from OctoPrint connected to the mainboard, from ESP3D WiFi module connected to the mainboard, from ESP3D WiFi module connected to the TFT, etc, etc, etc)

> when i'm starting to print the status line disappear, and instead of it there is progress bar

When print is started the TFT switches to a new menu, the "Printing" menu, hence the new layout of what's on the TFT.

>How can i make the extruder position and the layer height to be visible during printing

There is no configurable option for the "extruder position" to be visible during print, you must rewrite the source code for your own personal needs.

For the layer height issue more data needed for an answer to be given. The "Layer height" icon is showing the layer height of the currently  printed object. Just as you enter into the "Printing" menu, there is no printed object yet, the print process just have started, nothing printed yet. From your description it is unclear if the display of "---" is persistent or not, it is unclear if when the actual print starts the layer number is displayed or not. 

Even if I answered here, this section ("Discussions") is a place for generic talk, for sharing ideas, etc. It seems you have some problems, you do not want to discuss something general so you would have a better chance of solving the issues you have in the "Issues" section of this repository.

**Reply by alxsfrm** [reply]
Hi @kisslorand,
Thank you for the reply.
I'm printing from onboard SD.
The display of --- is persistent during the entire print.
Sorry about wrong section, moving to Issues.
